MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,279 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
 {% endhighlight %}
 {% endtabs %}
 
+# Show and Hide form fields 
```

Comment:
The header structure is incorrect. These represents the heading styles. Please understand its purpose in md files and use correctly

<img width="516" alt="image" src="https://github.com/user-attachments/assets/6089c1f2-f42c-4cd8-9994-d4b4c3c847aa" />